src/test/java/hudson/plugins/robot/RobotParserTest.java

```diff
@@ -183,6 +183,12 @@ public void testRobot4If() {
         final String dir = ".";
         final String mask = "robot4_if_output.xml";
         parse(dir, mask);
+    }
 
+    @Test
+    public void testRobot5Basic() {
+        final String dir = "robot5";
+        final String mask = "basic_output.xml";
```

Comment:
The same problem as above: we won't remember in the future why we have `basic_output.xml` for RF5, rather than have a more descriptive name eg. `robot5_except_while_output.xml`

Reply:
Renamed output file.